sdk/src/wallet/operations/transaction/input_selection.rs

```diff
@@ -38,6 +38,7 @@ where
         #[cfg(feature = "participation")]
         let voting_output = self.get_voting_output().await?;
         let protocol_parameters = self.client().get_protocol_parameters().await?;
+        let creation_slot_index = self.client().get_slot_index().await?;
```

Comment:
Shouldn't this rather get the slot index from the issuance commitment? Otherwise there could maybe a mismatch?
So `let creation_slot_index = slot_commitment_id.slot_index();`

Reply:
That was the issue causing the mana burn. The commitment ID is the latest ID but the slot index is more recent.